Packs/CortexCloud/Playbooks/Cortex_Cloud_-_AWS_EC2_Instance_Misconfigurations_Remediation.yml

```diff
@@ -0,0 +1,730 @@
+        step, skip the creation of a ticket and notify only, or choose to create a
+        ticket and notify stakeholders. The playbook checks for and requires existing
+        Jira, ServiceNOW, Slack, or MS Teams integrations to perform the ticketing
+        and notification actions. If none of these integrations are found in your
+        account, then the playbook sends an email with the issue details to the selected
+        issue owner. '
+      playbookName: Cortex Cloud - Create Ticket and Notify
+      type: playbook
+      iscommand: false
+      brand: ""
+      playbooktaskmissingcomponent: null
+    nexttasks:
+      '#none#':
+      - "20"
+    scriptarguments:
+      Asset details:
+        simple: ${Core.CoreAsset}
+      Issue State:
+        simple: Update
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 60,
+          "y": 1490
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "22":
+    id: "22"
+    taskid: a75584fe-eec6-4090-8b82-8e1067d9f828
+    type: condition
+    task:
+      id: a75584fe-eec6-4090-8b82-8e1067d9f828
+      version: -1
+      name: Check selected remediation action
+      type: condition
+      iscommand: false
+      brand: ""
+      playbooktaskmissingcomponent: null
+    nexttasks:
+      '#default#':
+      - "19"
+      Execute:
+      - "23"
+      Review:
+      - "11"
+    separatecontext: false
+    conditions:
+    - label: Execute
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Execute remediation or Assign for a review ?.Answers.0
+            iscontext: true
+          right:
+            value:
+              simple: Execute
+    - label: Review
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: Execute remediation or Assign for a review ?.Answers.0
+            iscontext: true
+          right:
+            value:
+              simple: Review
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Execute remediation or Assign for a review ?.Answers.name
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 60,
+          "y": 890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "23":
+    id: "23"
+    taskid: ab943443-4877-4d49-a92a-af74b103cd1f
+    type: regular
+    task:
+      id: ab943443-4877-4d49-a92a-af74b103cd1f
+      version: -1
+      name: aws-ec2-instance-metadata-options-modify
```

Comment:
Playbook task names should indicate what action they perform instead of simply listing the called script/playbook name.